### PR TITLE
Feature/jjdd8 ism 12

### PR DESCRIPTION
**Pull request by RobertHilla**
Adding and removing Events

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/App.java [hunk 1]
```diff
@@ -21,8 +22,11 @@ public static void main(String[] args) {
         Parser parser = new Parser();
         parser.initialization(JSON_REPOSITORY);
 
-        Menu.mainMenu();
-
+        //Menu.mainMenu();
+        //AddNewEvents addNewEvents = new AddNewEvents();
+        //addNewEvents.addNewEvent();
```
Comment (GregChyla) [inline src/main/java/com/infoshareacademy/App.java]: komentarze w kodzie

File: src/main/java/com/infoshareacademy/App.java [hunk 2]
```diff
@@ -21,8 +22,11 @@ public static void main(String[] args) {
...
+        RemoveEvents removeEvents = new RemoveEvents();
+        removeEvents.removeEvent(70023);
```
Comment (GregChyla) [inline src/main/java/com/infoshareacademy/App.java]: to ma być w głównej klasie uruchomieniowej?

File: src/main/java/com/infoshareacademy/menu/AddNewEvents.java [hunk 3]
```diff
@@ -29,7 +29,10 @@
     public final static String EMPTY_STRING = "";
```
Comment (GregChyla) [inline src/main/java/com/infoshareacademy/menu/AddNewEvents.java]: stała do pustego stringa?

File: src/main/java/com/infoshareacademy/menu/AddNewEvents.java [hunk 4]
```diff
@@ -29,7 +29,10 @@
...
     public final static Integer EMPTY_INTEGER = 0;
```
Comment (GregChyla) [inline src/main/java/com/infoshareacademy/menu/AddNewEvents.java]: j.w.

**Comment by GregChyla** [comment]
Git

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**